### PR TITLE
doc: fix typo in pkg

### PR DESCRIPTION
**Pull request by halfcrazy**
**What this PR does / why we need it**:
fix typo in pkg

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by hzxuzhonghu** [comment]
I think you should make your typos prs into one. This is a waste of ci resource.

**Comment by halfcrazy** [comment]
ok-to-test？

File: pkg/kubelet/server/server_test.go [hunk 1]
```diff
@@ -573,7 +573,7 @@ func TestHealthCheck(t *testing.T) {
 func assertHealthFails(t *testing.T, httpURL string, expectedErrorCode int) {
 	resp, err := http.Get(httpURL)
 	if err != nil {
-		t.Fatalf("Got error GETing: %v", err)
```
Comment (davidz627) [inline pkg/kubelet/server/server_test.go]: I dont think any of these "GETting" are typos

**Pull request closed without merging**